src/services/subgraph.ts

```diff
@@ -0,0 +1,118 @@
+import axios from 'axios';
+import {
+    SubgraphResponse,
+    SubgraphAPIResponse,
+    StratReportGraphResult,
+    StrategyWithReports,
+    VaultWithStrategies,
+} from '../types';
+
+export class Subgraph {
```

Comment:
New Subgraph class to pull data from subgraph. So far it pulls addresses for vaults & strats plus the Strategy Reports.